Android/PushTutorial/app/src/main/java/io/ably/example/androidpushexample/MainActivity.java

```diff
@@ -117,17 +140,21 @@ public void onConnectionStateChanged(ConnectionStateChange state) {
 
     }
 
+    private String getClientId() {
+        String clientId = Settings.Secure.getString(getContentResolver(), Settings.Secure.ANDROID_ID);
+        return clientId;
+    }
+
     private void initAblyPush() throws AblyException {
-        LocalBroadcastManager.getInstance(this).registerReceiver(pushReceiver, new IntentFilter("io.ably.broadcast.PUSH_ACTIVATE"));
         ablyRealtime.push.activate();
     }
 
     private void subscribeChannels() {
 
-        ablyRealtime.channels.get(TEST_PUSH_CHANNEL).push.subscribeClientAsync(new CompletionListener() {
+        ablyRealtime.channels.get(TEST_PUSH_CHANNEL_NAME).push.subscribeClientAsync(new CompletionListener() {
```

Comment:
Similarly, this change ought to be fixed in the initial commit, not corrected here